data/remnant/remnant news.txt

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2020 by Zitchas
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+phrase "00-07"
+	word
+		"00"
+		"01"
+		"03"
+		"04"
+		"05"
+		"06"
+		"07"
+
+
+
+phrase "00-07 days"
+	phrase
+		"00-07"
+	word
+		" days"
+
+
+
+phrase "0-F"
+	word
+		"0"
+		"1"
+		"2"
+		"3"
+		"4"
+		"5"
+		"6"
+		"7"
+		"8"
+		"9"
+		"A"
+		"B"
+		"C"
+		"D"
+		"E"
+		"F"
+
+
+
+phrase "00-1F"
+	word
+		"0"
+		"1"
+	phrase
+		"0-F"
+
+
+
+phrase "A group of"
+	word
+		"A "
+	word
+		"pair "
+		"couple "
+		"pack "
+		"squad "
+		"team "
+		"family "
+		"group "
+		"gathering "
+		"small group "
+		"medium group "
+		"large group "
+	word
+		"of"
+
+
+
+phrase "Indeterminate"
+	word
+		"A few "
+		"Several "
+		"Some "
+		"Lots of "
+		"Many "
+
+
+
+phrase "Amount of Remnant"
+	phrase
+		"A group of"
+		"Indeterminate"
+	word
+		"Remnant"
+
+
+news "remnant security advisory"
+	location
+		government "Remnant"
+	name
+		word
+			"System Security "
+			"Local "
+			"System "
+			"Defense "
+		word
+			"Advisory"
+			"Status"
+			"Threats"
```

Comment:
```suggestion
			"advisory"
			"status"
			"threats"
```

Reply:
This is intended to be in Title Case. Suggesting this has, however, brought to my attention the fact that I missed capitalizing "Security" in the set before it. Thanks!